checkstyle/suppressions.xml

```diff
@@ -98,6 +98,7 @@
     <suppress checks="CyclomaticComplexity" files="com/hazelcast/config/AbstractXmlConfigHelper"/>
     <suppress checks="ClassDataAbstractionCoupling" files="com/hazelcast/config/AbstractXmlConfigHelper"/>
     <suppress checks="FanOutComplexity" files="com/hazelcast/config/ConfigXmlGenerator"/>
+    <suppress checks="ClassFanOutComplexityCheck" files="com/hazelcast/config/Config"/>
 
```

Comment:
This is actually a regex matching any part of the name. For example, just `Complexity` would also word and exclude all complexity checks.